src/store/actions/test.js

```diff
@@ -32,5 +35,13 @@ describe( 'Store', () => {
 			expect( action.type ).toEqual( ActionTypes.BLOCK.DELETE );
 			expect( action.clientId ).toEqual( '1' );
 		} );
+
+		it( 'should create an action to create a block', () => {
+			registerCoreBlocks();
```

Comment:
This feels like a big thing to do while in a section of an otherwise simple test where we just test the fields of a newly created action object.

Can we perhaps mock the block to pass it to the action creation function?

On the other hand, judging from the convo we had over Slack, this will probably change when the block creation itself might move into the reducer so, perhaps we can leave this PR as is.

Reply:
That's a good observation - also this test for now tests for creation action type only but I think should check for each of the registered blocks if that makes sense (once the function actually creates a block instead of just carrying a block to be added/inserted).  
Added to my item list https://github.com/wordpress-mobile/gutenberg-mobile/issues/58#issuecomment-409571383

Reply:
I'm still not comfortable with `registerCodeBlocks` being inside the testcase since it's not the subject of the test itself. Let's move it to the `beforeAll` section so it's clear that it's part of the setup, not the test. WDYT?

Reply:
👍

Reply:
addressed in a04c5e9